### DISCUSSION TITLE
Github Pages and Jekyll theme

### DISCUSSION THREAD

**Discussion by ASandDA** [community/community#28704] [New to GitHub] [answered]
I am very new and trying to view the website I added to Github. At first the page would not publish at all. After reading through others who had this problem, I saw that I'm required to choose a theme. So, I chose a theme, and then I was finally able to see the Github Page. However, the page is only showing as the theme that I chose and has none of the content that I added to the repository. I tried deleting the theme, but that took me back to the page not existing.

Can anyone help me with this?
Thank you!

**Comment by radar-blip** [comment, marked as answer]
A theme is not required, its optional. All you need is your index.html file. Did you try that alone?

**Reply by ASandDA** [reply]
I just tried that but it gave the same issue. I'm going to try uploading to an entirely new repo again and see if it works.

**Reply by radar-blip** [reply]
Ok, good luck!

**Reply by ASandDA** [reply]
Unfortunately, that didn't work. I am back to the original 
![Screenshot 2022-08-07 231601](https://user-images.githubusercontent.com/110643351/183331500-d732b376-804d-4a29-bf4a-5f6c5682a1fc.png)
issue of the page not building.

**Reply by radar-blip** [reply]
It looks like its working now.

**Reply by ASandDA** [reply]
AHhh, the site is indeed working, although all my graphics seem to have been deleted. I can add those back in though. thank you SO SO MUCH!

**Reply by radar-blip** [reply]
You're welcome (although I didnt do much) anyway, good going and have a nice flight!

**Comment by jfullstackdev** [comment]
another thing you might want to consider is your repo name

![image](https://user-images.githubusercontent.com/47092464/183344470-986edd2f-830d-4c4b-94f1-9bf8e53de427.png)

as you can see, it's not at the root of your page, in order for you to have it at your root, simple change the repo name from

`Portfolio.github.io` to `ASandDA.github.io`

**Comment by ASandDA** [comment]
ok thank you so much for that advice!
